net-im/discord-bin/discord-bin-0.0.21-r1.ebuild

```diff
@@ -76,51 +79,81 @@ CONFIG_CHECK="~USER_NS"
 
 S="${WORKDIR}/${MY_PN^}"
 
-pkg_pretend() {
-	chromium_suid_sandbox_check_kernel_config
-}
-
 src_unpack() {
 	unpack ${MY_PN}-${MY_PV}.tar.gz
 }
 
 src_configure() {
-	chromium_suid_sandbox_check_kernel_config
-
 	default
+	chromium_suid_sandbox_check_kernel_config
 }
 
 src_prepare() {
 	default
 	# remove post-install script
 	rm postinst.sh || die "the removal of the unneeded post-install script failed"
 	# cleanup languages
-	pushd "locales/" || die "location change for language cleanup failed"
+	pushd "locales/" >/dev/null || die "location change for language cleanup failed"
 	chromium_remove_language_paks
-	popd || die "location reset for language cleanup failed"
+	popd >/dev/null || die "location reset for language cleanup failed"
 	# fix .desktop exec location
-	sed -i -e "s:/usr/share/discord/Discord:${DESTDIR}/${MY_PN^}:" ${MY_PN}.desktop || die "fixing of exec location on .desktop failed"
+	sed -i "/Exec/s:/usr/share/discord/Discord:${DESTDIR}/${MY_PN^}:" \
+		"${MY_PN}.desktop" ||
+		die "fixing of exec location on .desktop failed"
+	# USE seccomp
+	if ! use seccomp; then
+		sed -i '/Exec/s/Discord/Discord --disable-seccomp-filter-sandbox/' \
+			"${MY_PN}.desktop" ||
+			die "sed failed for seccomp"
+	fi
+	# USE system-ffmpeg
+	if use system-ffmpeg; then
+		rm libffmpeg.so || die
+		elog "Using system ffmpeg. This is experimental and may lead to crashes."
+	fi
+	# USE swiftshader
+	if ! use swiftshader; then
+		rm -r libvk_swiftshader.so swiftshader || die
+		elog "Running without SwiftShader OpenGL implementation. If Discord doesn't start "
+		elog "or you experience graphic issues, then try with USE=swiftshader enabled."
+	fi
```

Comment:
Again, this doesn't seem like a good idea. What's the point behind this?

I like this PR otherwise and it seems to work fine.

Reply:
@juippis To be perfectly honest, I don't have a valid technical reason for doing this; other than doing it because [the MS Team ebuild does it](https://github.com/gentoo/gentoo/blob/b045d231c0a10b9e9a46add0480d6c8fe2f8c77c/net-im/teams/teams-1.5.00.10453-r1.ebuild#L99).

I can remove this, though...